qgis_plugin_gpq_downloader.py

```diff
@@ -661,36 +662,54 @@ def run(self, default_source=None):
         # Connect validation complete signal to handle the result
         dialog.validation_complete.connect(
             lambda success, message, results: self.handle_validation_complete(
-                success, message, results, dialog.get_url(), self.iface.mapCanvas().extent()
+                success, message, results, dialog.get_url(), self.iface.mapCanvas().extent(), dialog
             )
         )
         
         if dialog.exec_() != QDialog.Accepted:
             return
 
-    def handle_validation_complete(self, success, message, validation_results, dataset_url, extent):
-        """Handle validation completion and start download if successful"""
+    def handle_validation_complete(self, success, message, validation_results, url, extent, dialog):
+        """Handle validation completion and start download if successful."""
         if success:
-            # Generate default filename
-            from datetime import datetime
-            timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
-            default_filename = f"geoparquet_download_{timestamp}.parquet"
-            default_save_path = str(self.download_dir / default_filename)
+            # Get current date for filename
+            current_date = datetime.datetime.now().strftime('%Y%m%d')
```

Comment:
Curious why you shortened this to just be year, month, day. Just to make it a bit shorter? 

I think I could be convinced of it, but the benefit of hour, minute, second is that if I download more than one file of one type I can just not think / edit. But if I pull down overture_buildings twice then it'll ask me to enter another name if it's still the same day.

Reply:
This was done in error. Makes sense to keep it as a full timestamp. I've reverted to the original.